pkg/tbtc/signing_test.go

```diff
@@ -202,6 +342,252 @@ func TestSigningRetryLoop(t *testing.T) {
 					lastAttempt,
 				)
 			}
+
+			if test.expectedLastAttempt != nil {
+				expectedOutgoingAnnouncementsCount :=
+					int(test.expectedLastAttempt.number)
+				actualOutgoingAnnouncementsCount :=
+					len(announcer.outgoingAnnouncements)
+				if expectedOutgoingAnnouncementsCount !=
+					actualOutgoingAnnouncementsCount {
+					t.Errorf(
+						"unexpected outgoing announcements count\n"+
+							"expected: [%+v]\n"+
+							"actual:   [%+v]",
+						expectedOutgoingAnnouncementsCount,
+						actualOutgoingAnnouncementsCount,
+					)
+				}
+
+				for attemptNumber, outgoingAnnouncement := range announcer.outgoingAnnouncements {
+					if test.signingGroupMemberIndex !=
+						outgoingAnnouncement.signingGroupMemberIndex {
+						t.Errorf(
+							"unexpected outgoing announcement's member "+
+								"index for attempt [%v]\n"+
+								"expected: [%+v]\n"+
+								"actual:   [%+v]",
+							attemptNumber,
+							test.signingGroupMemberIndex,
+							outgoingAnnouncement.signingGroupMemberIndex,
+						)
+					}
+
+					if message.Cmp(outgoingAnnouncement.message) != 0 {
+						t.Errorf(
+							"unexpected outgoing announcement's message "+
+								"for attempt [%v]\n"+
+								"expected: [%+v]\n"+
+								"actual:   [%+v]",
+							attemptNumber,
+							message,
+							outgoingAnnouncement.message,
+						)
+					}
```

Comment:
Could we use `testutils.AsertIntsEqual`?

Reply:
See https://github.com/keep-network/keep-core/pull/3361/commits/771b8707781a69eccf1dd5adaba3a712f75c3b78